packages/datagovmy-ui/src/lib/constants.ts

```diff
@@ -241,12 +252,106 @@ export const AgencyLink: Record<Agency, string> = {
   "moh": "https://www.moh.gov.my",
   "mot": "https://www.mot.gov.my/en/",
   "ntrc": "https://www.dermaorgan.gov.my/ntrc",
+  "npra": "https://www.npra.gov.my/index.php/en/",
+  "nres": "https://www.nres.gov.my/",
+  "parlimen": "https://www.parlimen.gov.my/",
+  "penjara": "https://www.prison.gov.my/",
+  "perhutanan": "https://www.forestry.gov.my/my",
+  "perikanan": "https://www.dof.gov.my",
   "pdn": "https://pdn.gov.my/v2/",
   "pdrm": "https://www.rmp.gov.my/",
   "perkeso": "https://www.perkeso.gov.my/en/",
   "phcorp": "https://protecthealth.com.my",
   "prasarana": "https://www.prasarana.com.my/",
+  "sesb": "https://www.sesb.com.my/",
+  "span": "https://www.span.gov.my/",
+  "st": "https://www.st.gov.my/",
+  "swk-energy": "https://www.sarawakenergy.com/",
   "spr": "https://www.spr.gov.my/",
   "ssm": "https://www.ssm.com.my/Pages/Home.aspx",
+  "tnb": "https://www.tnb.com.my/",
   "unhcr": "https://www.unhcr.org/my/",
 } as const;
+
+export const SOURCE_TRANSLATIONS: Record<string, { "ms-MY": string; "en-GB": string }> = {
```

Comment:
since we have agency name translation in `meta` repo, why is this needed here ?

Reply:
Mapped the key to full translation. Earlier was told not to touch the BE part.

Reply:
its not the BE part. its the `meta` part.

Reply:
ok